tools/format.py

```diff
@@ -48,24 +48,15 @@ def main():
         did_fmt = True
 
     if not did_fmt:
-        prettier()
+        dprint()
         yapf()
         rustfmt()
 
 
-def prettier():
-    script = os.path.join(third_party_path, "node_modules", "prettier",
-                          "bin-prettier.js")
-    source_files = get_sources(root_path, ["*.js", "*.json", "*.ts", "*.md"])
-    if source_files:
-        max_command_length = 24000
-        while len(source_files) > 0:
-            command = ["node", script, "--write", "--loglevel=error", "--"]
-            while len(source_files) > 0:
-                command.append(source_files.pop())
-                if len(" ".join(command)) > max_command_length:
```

Comment:
Interestingly, this `max_command_length` was being hit so there are 240 files not being formatted in Deno's codebase right now.

(I was wondering why I was getting a bunch of files that looked like they'd never been run through a formatter before.)

Reply:
I misread the code here so it's not 240 files, but it is a handful of files as if it pops from `source_files` and exceeds `max_command_length` then that popped file is not formatted.